rfcs/0021-glossary-terms.md

```diff
@@ -0,0 +1,123 @@
+## Summary
+[summary]: #summary
+
+This RFC proposes standardized glossary terms for various concepts in the Coda Protocol and how they are communicated to the external world in documentation, logging, user interfaces, and public communications. This is not intended to cover the existing codebase in scope, as it may be too cumbersome at this point to rename everything to fit these proposed conventions. 
+
+However, it is advisable that over time the codebase also migrates to this shared terminology so that future community members can on-board seamlessly from high level documentation to contributing code.
+
+## Motivation
+
+[motivation]: #motivation
+
+Currently, multiple terms exist for several concepts in the Coda protocol, and they are used interchangeably when communicating to the world. This has some downsides as there is a lack of clarity on which terms to use, and could potentially cause confusion for people unfamiliar with the protocol. 
+
+As such, the motivation of this RFC is to standardize the naming convention for various concepts into a unified glossary that can be used in documentation, user-facing copy, and marketing channels. The expected outcome is unified language when communicating to the public and in developing Coda related products. This can then lead to more clarity and easier onboarding.
+
+## Detailed design
+
+[detailed-design]: #detailed-design
+
+The proposal contains the suggested term with it's associated concept it encapsulates, as well as alternative terms and the rationale for  the suggestion.
+
+The terms proposed are below:
+
+### Block Producer
+**Concept:** A consensus node in the network that is selected for a slot, and generates a block.
```

Comment:
Suggest: "A consensus node in the network which is able to generate blocks and participates in a process to determine what blocks it is allowed to produce"